pinot-core/src/test/java/org/apache/pinot/queries/BigDecimalQueriesTest.java

```diff
@@ -387,54 +387,6 @@ public void testQueries() {
         assertEquals(row[0], BASE_BIG_DECIMAL.add(BigDecimal.valueOf(69)));
       }
     }
-    {
```

Comment:
Why do we remove these 2 tests?

Reply:
I removed proper null handling logic in equality/inequality/membership operators from this PR (based on your recommendation to simplify the PR), and as a result removed these test.

Reply:
I don't get it. This test already exists without the null support on filter, and should still work now right?

Reply:
@Jackie-Jiang We insert nulls in bigDecimalColumn (see line 112 in this file) and in the PR we handle nulls properly in aggregation functions including max. maxValue takes the value of null for some input. However, we removed the code that handle nulls in comparison (maxValue < %s) from this PR. As a result, I disabled the test.

Reply:
Oh, now I understand. The problem is from the having clause trying to match a `null` value. Shall we comment it out for now and add it back when the filter support is added?
You may also remove it for now and add it back in the next PR.

Reply:
This is the plan, to remove now and add it as part of the PR handling nulls in. equality/inequality/membership operators.